lib/fastlane/plugin/ddg_apple_automation/actions/asana_upload_action.rb

```diff
@@ -51,7 +48,7 @@ def self.available_options
                                        optional: false,
                                        type: String),
           FastlaneCore::ConfigItem.new(key: :file_name,
-                                       description: "Path to the file that will be uploaded",
+                                       description: "Path to a file that will be uploaded",
```

Comment:
I'm no expert but was thinking this parameter indicates a specific file, not any file :)